code/processes/metrics.q

```diff
@@ -36,16 +42,14 @@ metrics:{[syms]
    if[.metrics.enableallday;
     if[not all syms in key .metrics.start;.metrics.start::exec first time by sym from sumstab];
     t:`sym`timediff xasc t,select sym,timediff:0Nn,vwap:sumsps%sumssize,twap:sumspricetimediff%.z.p - .metrics.start[sym] from latest where sym in syms];
-
-   t
-  
+   t  
  }
 
 \d .metrics 
 
 / check for TP connection
 notpconnected:{[]
-	0 = count select from .sub.SUBSCRIPTIONS where proctype in ((),`tickerplant), active}
+	0 = count select from .sub.SUBSCRIPTIONS where proctype in tickerplanttypes,active}
 
 / get handle for TP & subscribe
 subscribe:{
```

Comment:
The if-statement in this function is not correctly indented, could fix it up while you're here.